apps/web/src/pages/Users/AdminUserAdvancedToolsPage/components/ActivityDetails.tsx

```diff
@@ -0,0 +1,67 @@
+import { defineMessage, useIntl } from "react-intl";
+import CalendarDaysIcon from "@heroicons/react/24/outline/CalendarDaysIcon";
+
+import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
+import { Card, TableOfContents } from "@gc-digital-talent/ui";
+import { commonMessages } from "@gc-digital-talent/i18n";
+
+import FieldDisplay from "~/components/FieldDisplay/FieldDisplay";
+
+export const ACTIVITY_DETAILS_ID = "activity-details";
+
+export const activityDetailsTitle = defineMessage({
+  defaultMessage: "Activity details",
+  id: "QXIC/M",
+  description: "Title of admin user activity details section",
+});
+
+const UserActivityDetails_Fragment = graphql(/** GraphQQL*/ `
+  fragment UserActivityDetails on User {
+    authInfo {
+      createdDate
+      updatedDate
+      lastSignInDate
+    }
+  }
+`);
+
+interface ActivityDetailsProps {
+  query: FragmentType<typeof UserActivityDetails_Fragment>;
+}
+
+const ActivityDetails = ({ query }: ActivityDetailsProps) => {
+  const intl = useIntl();
+  const user = getFragment(UserActivityDetails_Fragment, query);
+  const notAvailable = intl.formatMessage(commonMessages.notAvailable);
+
+  return (
+    <TableOfContents.Section id={ACTIVITY_DETAILS_ID}>
+      <TableOfContents.Heading
+        icon={CalendarDaysIcon}
+        color="secondary"
+        className="mb-6"
+      >
+        {intl.formatMessage(activityDetailsTitle)}
+      </TableOfContents.Heading>
+      <Card className="grid gap-6 sm:grid-cols-3">
+        <FieldDisplay label={intl.formatMessage(commonMessages.created)}>
+          {user.authInfo?.createdDate ?? notAvailable}
```

Comment:
I think you're displaying the raw UTC timestamps here.  We usually shift these to the user's local time zone before displaying, I think.